ftw/raven/config.py

```diff
@@ -32,6 +33,36 @@ def ignored_exception_classnames(self):
 
         return tuple(ignored)
 
+    @property
+    def tags(self):
+        value = self._get_tags_from_file()
+        value.update(self._get_tags_from_env())
```

Comment:
Combining them makes it possible to have server-wide tags and deployment-tags which are combined.
Your readme change proposal makes it clearer :wink: